src/Surfnet/StepupRa/RaBundle/Service/SecondFactorService.php

```diff
@@ -53,9 +53,10 @@ public function __construct(
      * @param string $registrationCode
      * @return null|VerifiedSecondFactor
      */
-    public function findVerifiedSecondFactorByRegistrationCode($registrationCode)
+    public function findVerifiedSecondFactorByRegistrationCode($registrationCode, $institution)
```

Comment:
Maybe it's better to rename `institution` `actorInstitution`.

Reply:
Yes, that makes sense, will fix that